ui-v2/app/adapters/application.js

```diff
@@ -44,15 +76,15 @@ export default Adapter.extend({
   getHost: function() {
     return this.host || `${location.protocol}//${location.host}`;
   },
-  slugFromURL: function(url) {
+  slugFromURL: function(url, decode = decodeURIComponent) {
     // follow the 'last part of the url is the id' rule
```

Comment:
Heh, this is the second time I've seen this comment - maybe worth calling out and elaborating in the README?

Reply:
Yeah, not sure how important it is for a README, maybe a CONTRIBUTING, actually I've been thinking of adding a CONTRIBUTING.md with my initial ember sketches in (remember those?) to illustrate the architecture, plus some other bits, what do you think? I could add things like that in there?

Reply:
For Vault we have a ui-specific README for that sort of thing - commands to build, how things are structured, etc.

Reply:
This one?

https://github.com/hashicorp/vault/blob/master/ui/README.md

Reply:
Yeah, it's ... shorter than I remember 😂, and that reminds me I've been meaning to document the model-form-framework-ish thing we do in Vault there 😅